### ISSUE TITLE
device_class missing from schema for KNX sensors

### ISSUE THREAD

**Issue by ehn** [keesschollaart81/vscode-home-assistant#2405]
The KNX integration lets you set `device_class` for sensors like so:

```YAML
knx:
  sensor:
    - name: Name of sensor
      state_address: '1/2/3'
      type: curr
      device_class: current
```

However, VS Code complains: `Property device_class is not allowed`.

**Comment by ehn** [comment]
It looks like #2397 would also fix this.

**Comment by frenck** [comment]
Resolved in #2397
